### PR TITLE
[13.0] Add delivery_postlogistics_dangerous_goods

### PR DESCRIPTION
**Pull request by mmequignon**
This modules overrides `Postlogistics Shipping` to declare dangerous goods
during postlogistics label generation.

Based on:
- https://github.com/OCA/delivery-carrier/pull/357

Supersedes:
- https://github.com/OCA/delivery-carrier/pull/342

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: delivery_postlogistics_dangerous_goods/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "complexity": "normal",
+    "depends": [
+        # OCA/delivery-carrier
+        "delivery_postlogistics",
+        # OCA/community-data-files
+        "l10n_eu_product_adr",
+    ],
+    "website": "https://github.com/OCA/delivery-carrier",
+    "installable": True,
+    "auto_install": False,
```
Comment (TDu) [inline delivery_postlogistics_dangerous_goods/__manifest__.py]: `auto_install` False is the default, so not really needed, here.
But does it not make sense to have it set to `True` ?

Comment (simahawk) [reply delivery_postlogistics_dangerous_goods/__manifest__.py]: @mmequignon ping

**Review by simahawk — APPROVED** [review]
LG, one comment by @TDu needs answer

File: delivery_postlogistics_dangerous_goods/__manifest__.py (hunk 1 above)
Comment (mmequignon) [reply delivery_postlogistics_dangerous_goods/__manifest__.py]: Yeah, I don't know in what case we should enable auto-install or not

Comment (TDu) [reply delivery_postlogistics_dangerous_goods/__manifest__.py]: With `auto_install` set to `True` your module will be automatically installed when both its dependencies are installed. Which in my opinion makes sense in this case.

Comment (sebalix) [reply delivery_postlogistics_dangerous_goods/__manifest__.py]: It makes sense to set it to `True` as this module doesn't break the base PostLogistics module.

**Comment by grindtildeath** [comment]
@mmequignon Can you please fix the CI ?

**Review by sebalix — COMMENTED** [review]
Don't mention `13.0` in the commit message as commits could be cherry-picked from one version to another.

**Comment by mmequignon** [comment]
> Don't mention `13.0` in the commit message as commits could be cherry-picked from one version to another.

done

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
Let's try again...
/ocabot merge nobump

**Comment by yvaucher** [comment]
Is it intended to have that kind of output? It seems too verbose and not giving significant informations. The label binary gives a lot of noise.
It could reach some limit of the output or something like that.

Travis is still unhappy.


https://app.travis-ci.com/github/OCA/delivery-carrier/jobs/543251740#L1416

**Comment by nilshamerlinck** [comment]
Hi @mmequignon we faced this issue before and a workaround was to mute vcr logger for the case involving the big response; I tried it [here](https://github.com/nilshamerlinck/delivery-carrier/commit/ffe84dde7239804792d7a9250a2d0d231610cf05), seems to work

**Comment by mmequignon** [comment]
@dreispt thanks Daniel, but I think there's some action to do on my side before trying.
I have no time right now, but will fine to find some.
Sorry about that…

**Comment by rousseldenis** [comment]
@mmequignon No news on this ?

**Comment by simahawk** [comment]
@mmequignon ping

**Comment by simahawk** [comment]
we keep only v14 https://github.com/OCA/wms/pull/334

**Pull request closed without merging**